New Rockplanet ruin, Budgetcuts!

spockye opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
This PR adds a Nanotrasen Research facility, in a bit of a pickle.

![2023 01 17-15 57 58](https://user-images.githubusercontent.com/79304582/212936680-642aae0c-15d2-48b6-8c75-724d12a2ad13.png)

![2023 01 17-16 16 30](https://user-images.githubusercontent.com/79304582/212936724-c2904f80-5558-49c4-bc2d-c63fd4ad70c5.png)

Loot:
Combat supplies ( 1 m1911 / energy crossbow / security locker / security vendor / hos locker )
Medical supplies ( compact defib / brute,burn,oxy kit ) 
Kitchen supplies
Research supplies ( sci protolathe / circuit imprinter / experimentor / slime cores / research note / nanite supplies )
Fake Death ripley
jackhammer

Threats:
Emergency Retirement Team ( Assault officer / 2x SMG officers / 2x PISTOL officers / 2x FIST officers )
Emergency Treasure Retrieval Team ( RANGED pirate / 2x SWORD pirates / HARDSUIT pirate ) 

Lore grimble!
As you know, Nanotrasen hasn't exactly had the most economically successful season. Due to this, theres a need for some.... Budget cuts _(get it? its the name! im really smart)_. This research station hasn't exactly been productive lately, nor has it been worth having a research station in a place as remote as this. Nanotrasen is now left with two choices. 
Pay the researchers their promised bonus, or... Well, you can guess the second option. 
The Research Director fearing what Nanotrasen would do to him sends out an SOS and kills himself.

Sadly for the rest of the crew, nobody responds to the SOS call, sealing their fate. A certain group of pirates, however, locate the signal and prepare to raid it. 
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

<!-- Tick the box below (put an X instead of a space between the brackets) if you have tested your changes and this is ready for review. Leave unticked if you have yet to test your changes and this is not ready for review. -->

- [x] I affirm that I have tested all of my proposed changes and that any issues found during tested have been addressed.

## Why It's Good For The Game
More ruins = good! more diversity in ruins is also good!
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog

:cl:
add: Added a new Rock ruin, rockplanet_budgetcuts
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

spockye committed:
main files

spockye committed:
misc changes

spockye committed:
more small changes

we love commit spam

spockye committed:
switched apc type out

from 15k to normal

spockye committed:
attempts to fix GPS SOS

Erikafox commented on code/datums/ruins/rockplanet.dm:
```diff
@@ -86,3 +86,9 @@
+
+/datum/map_template/ruin/rockplanet/budgetcuts
+	name = "Budgetcuts"
+	description = "Nanotrasen's gotta lay off some personell, and this facility hasn't been worth the effort so far"
```
personnel

Erikafox commented on code/game/area/areas/ruins/rockplanet.dm:
```diff
@@ -6,3 +6,11 @@
 
+//rockplanet_budgetcuts
+
+/area/mine/rockplanet
```
bad ruin area path, check the other ruin area .dms and repath accordingly.

Erikafox commented on _maps/map_catalogue.txt:
```diff
@@ -230,6 +230,10 @@ Find the key for using this catalogue in "map_catalogue_key.txt"
 	RockRuins:
+		File Name = "_maps\RandomRuins\RockRuins\rockplanet_budgetcuts.dmm"
+		Size = (x = 52)(y = 44)(z = 1)
+		Tags = "hard combat Challenge", "Medium Loot", "hospitable"
```
capitalize

Erikafox commented on _maps/RandomRuins/RockRuins/rockplanet_budgetcuts.dmm:
```diff
@@ -0,0 +1,5319 @@
+/obj/item/paper/crumpled/awaymissions/research,
+/obj/item/paper/fluff/awaymissions/moonoutpost19/research/evacuation,
+/obj/item/paper/crumpled/awaymissions/research/sensitive_info,
+/obj/item/paper/fluff/awaymissions/moonoutpost19/research/xeno_queen,
```
read these papers and make sure they don't go into bizarro world, they're old.

Erikafox reviewed (commented):
map review later

spockye committed:
misc fixes

fixed personnel

fixed capitalization

fixed zones

removed old notes

Erikafox reviewed (changes requested):
![image](https://user-images.githubusercontent.com/94164348/213476729-b5245eb2-1f8e-46fa-80a7-b588d970c4be.png)
I have the odd feeling that one of these guys will bump the door, they'll see each other, and they'll fight.
![StrongDMM_P5RZzlAnkq](https://user-images.githubusercontent.com/94164348/213477169-7702f774-82ed-49f3-a030-4453b4c94c01.png)
ditch the consuming and industrial extracts. Greys for xenobio is good loot already.
![StrongDMM_UhdhGNJq5O](https://user-images.githubusercontent.com/94164348/213477384-53d12f50-9c31-41d9-9d9e-8e061f3aaa36.png)
swap for a helm console, just have it be broken.
![StrongDMM_ceWQgYIS0q](https://user-images.githubusercontent.com/94164348/213477522-5398c3f6-c8b9-4617-af3f-5cbb7409dcc6.png)
Swap the explosive mine for a (sophisticated) shrapnel mine. Explosives are not really fun to deal with. Shrapnel has a bit more grace.
I also wouldn't be upset if you added more.
![StrongDMM_dItupkaFek](https://user-images.githubusercontent.com/94164348/213477851-1af18ac3-f4bb-4e80-960d-0d8889e5790d.png)
what's up with this?
![StrongDMM_1kAI6Y4Q8N](https://user-images.githubusercontent.com/94164348/213477986-9b19cf26-47c1-41ad-9e19-60728e4b50a3.png)
Decal

spockye committed:
misc changes

removes old mines, replaces with shrapnel (non crashy type)

misc decal additions

barricades Hos office door

removes consuming and industrial slime cores

spockye committed:
Merge branch 'master' into new-ruin-1

Erikafox commented:
The pirates outside should be the suited variants, they'll die otherwise.

Erikafox committed:
Merge branch 'master' into new-ruin-1

Signed-off-by: Bjarl <94164348+Bjarl@users.noreply.github.com>

Erikafox commented:
![firefox_YSfXdEnTJL](https://user-images.githubusercontent.com/94164348/219689363-8a6488ff-6048-433e-bad1-0c8475a5f749.png)
The real authentic HoS locker needs to go, it has a secfab and enough gear for a small ship.

Erikafox committed:
Merge branch 'master' into new-ruin-1

spockye committed:
Switches Hos locker

Erikafox commented:
Need to fix the pirates on the outside.

Erikafox committed:
Merge branch 'master' into new-ruin-1

Erikafox commented:
fix the pirates

Erikafox commented:
re-open when you fix the pirates.

spockye commented:
GWA

spockye committed:
Merge branch 'master' into new-ruin-1

Erikafox committed:
Merge branch 'master' into new-ruin-1
